near-sdk/src/collections/mod.rs

```diff
@@ -59,7 +59,7 @@ pub use lazy_option::LazyOption;
 mod tree_map;
 pub use tree_map::TreeMap;
 
-pub const ERR_INCONSISTENT_STATE: &str = "The collection is an inconsistent state. Did previous smart contract execution terminate unexpectedly?";
+pub const ERR_INCONSISTENT_STATE: &str = "The collection is an inconsistent state. Have any collections been updated without updating themselves in state?";
```

Comment:
Think this should read better:
```suggestion
pub const ERR_INCONSISTENT_STATE: &str = "The collection is in an inconsistent state. Have any collections been updated without committing changes to contract state?";
```